src/Libraries/Microsoft.Extensions.Resilience/Microsoft.Extensions.Resilience.csproj

```diff
@@ -31,7 +31,6 @@
     <PackageReference Include="Polly.Core" />
     <PackageReference Include="Polly.Extensions" />
     <PackageReference Include="Polly.RateLimiting" />
-    <PackageReference Include="Microsoft.Bcl.HashCode" Condition="!$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net6.0'))" />
```

Comment:
I don't think this is right. This package should still be needed in some configurations.

Reply:
Which case are you worried about?

The code builds, so it seems to be working.